test/integration/helpers/testHelpers.ts

```diff
@@ -0,0 +1,36 @@
+import { readFileSync } from 'fs';
+import { DetailedPopulationGroupResult, ExecutionResult, PopulationResult } from '../../../src/types/Calculator';
+import { PopulationType } from '../../../src/types/Enums';
+
+/**
+ * Retrieves JSON fixture from the integration directory for easier ts casting in test files
+ */
+export function getJSONFixture(path: string): any {
+  return JSON.parse(readFileSync(`test/integration/${path}`).toString());
+}
+
+/**
+ * Formats expectedPopulations object into an array of objects which can be passed into an
+ * expect.arrayContaining and compared with result
+ */
+export function assertPopulationResults(
+  result: ExecutionResult<DetailedPopulationGroupResult>,
+  expectedPopulations: Partial<Record<PopulationType, boolean>>
+) {
+  const expectedPopulationResults: PopulationResult[] = Object.entries(expectedPopulations).map(([popType, value]) =>
```

Comment:
good catch! Just so I understand, you're asking for the `results` arg to be a bit more specific here? Do you think it's a good plan to parse through the measure on the test side to grab the groupId and then just drill down to the specific group and pass that in for results instead?

Reply:
Good point. Maybe this helper should just take the results for a specific group (i.e. the `expect` starts on an entry of the `detailedResults` list rather than doing an expect.arraycontaining on it

Reply:
^ I think this is the better way, even though it will push more logic into the actual test. Although I wonder having a helper to grab the population group results by index or id and throw an `expect` if it cannot be found. Then you can take it and pass into this function to compare results.